codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/HttpBoundProtocolGenerator.kt

```diff
@@ -67,7 +67,7 @@ class HttpBoundProtocolGenerator(
     HttpBoundProtocolTraitImplGenerator(codegenContext, protocol),
```

Comment:
I will be using the code generated by this file in the server codegen to support some of the ser/de tests I am generating. For this reason I need to make some visibility changes to this class. It is not a strict contract since the code generated is used only in tests.